README.md

```diff
@@ -91,6 +92,10 @@ Note: Our setup script does these two for you, but they are here for reference.
     ```bash
     ./setup.sh
     ```
+1. Run docker
+   ```bash
+   docker start [options] [container_name/container_id]
+   ```
```

Comment:
This is the command to actually start running a docker container.

I thought the issue was the Docker Daemon / Docker Desktop not running. 

We need an action item here that would prevent this check from failing:
https://github.com/Skyvern-AI/skyvern/blob/999eda9b5d750af6989ccbc9b4ef0a73f7ee8582/setup.sh#L199-L203

Reply:
Yes, exactly! 

If you could tell how people can run it for each OS that would be perfect. I know it's not always easily possible from the CLI. So for windows you could just say run Docker Desktop, etc.